mk/config.mk

```diff
@@ -298,6 +298,10 @@ CFG_REE_FS_TA ?= y
 CFG_REE_FS_TA_BUFFERED ?= n
 $(eval $(call cfg-depends-all,CFG_REE_FS_TA_BUFFERED,CFG_REE_FS_TA))
 
+# Allow REE FS content to be reset. If .../tee root directory does not
+# exists, assume it is virgin even of an anti rollback hask exists.
```

Comment:
OK, I would also mention RPMB and the real path (even if it can be changed and is different for AOSP):
```
# When CFG_REE_FS=y and CFG_RPMB_FS=y:
# Allow secure storage in the REE FS to be entirely deleted without causing
# anti-rollback errors. That is, rm -rf /data/tee (or whatever path is
# configured in tee-supplicant as CFG_TEE_FS_PARENT_PATH) can be used to reset
# the secure storage to a clean, empty state.
# Typically used for testing only since it weakens storage security.
```

Reply:
Slightly changed:
```
# When CFG_REE_FS=y and CFG_RPMB_FS=y:
# Allow secure storage in the REE FS to be deleted without causing
# anti-rollback errors. That is, rm -rf /data/tee/dirf.db (or whatever path is
# configured in tee-supplicant as CFG_TEE_FS_PARENT_PATH) can be used to reset
# the secure storage to a clean, empty state.
# Typically used for testing only since it weakens storage security.